lib/grammers-client/src/client/client.rs

```diff
@@ -52,6 +53,7 @@ pub struct InitParams {
     // TODO catch up doesn't occur until we get an update that tells us if there was a gap, but
     // maybe we should forcibly try to get difference even if we didn't miss anything?
     pub catch_up: bool,
+    pub server_addr: Option<SocketAddr>,
```

Comment:
Could you add a doccomment on top of this field to explain what it does and why anyone would want to use it?

Reply:
Yep